src/lib/Scenes/MyCollection/Screens/ArtworkForm/Form/artworkSchema.tsx

```diff
@@ -1,12 +1,15 @@
 import { yupToFormErrors } from "formik"
 import { ArtworkFormValues } from "lib/Scenes/MyCollection/State/MyCollectionArtworkModel"
+import { trim } from "lodash"
 import * as Yup from "yup"
 
 export const artworkSchema = Yup.object().shape({
   artistSearchResult: Yup.object()
     .nullable()
     .test("artistSearchResult", "Artist search result required", (value) => value !== null),
-  title: Yup.string().required("Title is required"),
+  title: Yup.string()
+    .required("Title is required")
+    .test("title", "Title should not be empty", (value) => trim(value || "") !== ""),
```

Comment:
Would this work as well? Just wondering if this would be easier to read 🤔 

```suggestion
    .test("title", "Title should not be empty", (value) => value && trim(value)),
```

Reply:
```diff
-    .test("title", "Title should not be empty", (value) => value && trim(value)),
+    .test("title", "Title should not be empty", (value) => !!value && trim(value) !== ""),
``` 
this one would work, because the test function requires a boolean, and its cleaner in my opinion, thanks for the suggestion 👌

Reply:
looks great!